src/Schema/Events/Click.ts

```diff
@@ -1812,21 +1812,25 @@ export interface ClickedNavBar {
  *  ```
  *  {
  *    action: "clickedUploadArtwork",
- *    context_module: "alerts-price"
- *    context_page_owner_type: "demand",
- *    context_page_owner_id: "",
+ *    alert_page_rank: 1,
  *    artist_id: "4212691337420",
- *    search_criteria_id: "4212691337420"
+ *    context_module: "alerts-price",
+ *    context_page_owner_id: "",
+ *    context_page_owner_type: "demand",
+ *    partner_search_criteria_id: "123abc",
+ *    search_criteria_id: "4212691337420",
  *    subject: "Upload Artworks"
  *  }
  * ```
  */
 export interface ClickedUploadArtwork {
   action: ActionType.clickedUploadArtwork
+  alert_page_rank?: number
+  artist_id?: string
   context_module: ContextModule
-  context_page_owner_type: PageOwnerType
   context_page_owner_id?: string
-  artist_id?: string
+  context_page_owner_type: PageOwnerType
+  partner_search_criteria_id?: string
   search_criteria_id?: string
   subject: string
```

Comment:
Just to clarify for me, we do still need both `partner_search_criteria_id` and `search_criteria_id`?

Reply:
Technically we probably don't need `search_criteria_id` but I don't think it hurts anything. It's also the case that past events will have this id in segment so idk if there are implications there? /cc @leodmz

Reply:
🙏🏻 up to you guys!

Reply:
coming back to this, if past events have this info I think we have to keep it to avoid breaking anything downstream